.github/workflows/causal_lm_cpp.yml

```diff
@@ -459,6 +469,7 @@ jobs:
           source ./ov/setupvars.sh
           python -m pip install --upgrade-strategy eager -r ./text_generation/causal_lm/cpp/requirements.txt
           python -m pip install ./thirdparty/openvino_tokenizers/[transformers]
+          sudo apt-get install libtbb-dev
```

Comment:
This would make TBB optional, `speculative_decoding_lm` will compile but fail in runtime without TBB found. Do I need to implement a fallback to single threaded for loop?